dev-python/h5py/h5py-2.10.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit distutils-r1 flag-o-matic
+
+DESCRIPTION="Simple Python interface to HDF5 files"
+HOMEPAGE="https://github.com/h5py/h5py https://www.h5py.org"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc test examples mpi"
+
+RDEPEND="
+	sci-libs/hdf5:=[mpi=,hl(+)]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]"
+
+BDEPEND="dev-python/pkgconfig"
+
+DEPEND="${RDEPEND}
+	dev-python/cython[${PYTHON_USEDEP}]
+	mpi? ( dev-python/mpi4py[${PYTHON_USEDEP}] )
+	doc? ( dev-python/alabaster[${PYTHON_USEDEP}] )
+	test? ( dev-python/PyQt5[testlib,${PYTHON_USEDEP}] )"
+
+distutils_enable_tests setup.py
+distutils_enable_sphinx docs --no-autodoc
+
+pkg_setup() {
+	use mpi && export CC=mpicc
+}
+
+python_prepare_all() {
+	append-cflags -fno-strict-aliasing
+	distutils-r1_python_prepare_all
+}
+
+python_configure() {
+	esetup.py configure $(usex mpi --mpi '')
+}
+
+python_install_all() {
+	DOCS=( README.rst AUTHORS ANN.rst )
+	use examples && DOCS+=( examples )
```

Comment:
It should be `local DOCS` but I don't remember if it worked or not.

Reply:
Are you sure? It says on the [wiki](https://devmanual.gentoo.org/ebuild-writing/variables/) that `DOCS` is a variable of the same type as `EAPI` `DESCRIPTION` etc.

Also, distutils-r1.eclass seems to imply that it should be without `local`:
```
# @ECLASS-VARIABLE: DOCS
# @DEFAULT_UNSET
# @DESCRIPTION:
# An array containing documents installed using dodoc. The files listed
# there must exist in the directory from which
# distutils-r1_python_install_all() is run (${S} by default).
#
# If unset, the function will instead look up files matching default
# filename pattern list (from the Package Manager Specification),
# and install those found.
#
# Example:
# @CODE
# DOCS=( NEWS README )
# @CODE
```

Reply:
You can define DOCS globally yes as in the above example, but you shouldn't modify global variables inside a phase. Hence `local DOCS`. It's a bash thing.
https://projects.gentoo.org/pms/7/pms.html#x1-11400011.2
https://projects.gentoo.org/pms/7/pms.html#x1-600007.1